src/main/java/ui/FontFrame.java

```diff
@@ -0,0 +1,137 @@
+package ui;
+
+import java.awt.BorderLayout;
+import java.awt.EventQueue;
+
+import javax.swing.JFrame;
+import javax.swing.JPanel;
+import javax.swing.border.EmptyBorder;
+import javax.swing.JButton;
+import javax.swing.BoxLayout;
+import java.awt.GridLayout;
+import java.awt.event.ActionListener;
+import java.awt.event.WindowAdapter;
+import java.awt.event.WindowEvent;
+import java.awt.event.ActionEvent;
+import javax.swing.JLabel;
+import javax.swing.JList;
+import javax.swing.AbstractListModel;
+import javax.swing.JSpinner;
+import javax.swing.JTextPane;
+import javax.swing.JComboBox;
+import javax.swing.DefaultComboBoxModel;
+import java.awt.Font;
+
+public class FontFrame extends JFrame {
+
+	private JPanel contentPane;
+	
+	JComboBox familyChooseBox;
+	JComboBox styleChooseBox;
+	JComboBox sizeChooseBox;
+
+	/**
+	 * Launch the application.
+	 */
+	/*
+	public static void main(String[] args) {
+		EventQueue.invokeLater(new Runnable() {
+			public void run() {
+				try {
+					FontFrame frame = new FontFrame();
+					frame.setVisible(true);
+				} catch (Exception e) {
+					e.printStackTrace();
+				}
+			}
+		});
+	}
+		*/
+	/**
+	 * Create the frame.
+	 */
+	public FontFrame(JTextPane currentPane) {
+		setDefaultCloseOperation(JFrame.EXIT_ON_CLOSE);
+		setBounds(100, 100, 296, 312);
+		contentPane = new JPanel();
+		contentPane.setBorder(new EmptyBorder(5, 5, 5, 5));
+		setContentPane(contentPane);
+		contentPane.setLayout(null);
+		
+		JButton yesButton = new JButton("Yes");
+		yesButton.addActionListener(new ActionListener() {
+			public void actionPerformed(ActionEvent e) {
+				String family = familyChooseBox.getSelectedItem().toString();
+				int size = Integer.parseInt(sizeChooseBox.getSelectedItem().toString());
+				String style = styleChooseBox.getSelectedItem().toString();
+				
+				 switch (style) {
```

Comment:
1. Code style here is wired. Can you please adjust it.
2. Add default to handle unexpected exception.